### DISCUSSION TITLE
Mobile App Support

### DISCUSSION THREAD

**Discussion by zenof** [vslinko/obsidian-outliner#170] [Archive]
Obsidian has already release the [mobile apps](https://obsidian.md/mobile) to public. However, in the mobile app it could not install the outliner plugin since the `isDesktopOnly` attribute is set to True in manifest file.

**Comment by vslinko** [comment]
@zenof Thanks for your feedback! Mobile version is already under discussion in other threads.
https://github.com/vslinko/obsidian-outliner/discussions/81
https://github.com/vslinko/obsidian-outliner/discussions/124

**Reply by zenof** [reply]
I am sorry for duplicate, I didn't found these topics before. Thank you!
